src/uct/ib/mlx5/gga/gga_mlx5.c

```diff
@@ -105,7 +104,8 @@ uct_gga_mlx5_rkey_unpack(uct_component_t *component, const void *rkey_buffer,
     }
 
     rkey_handle->packed_mkey    = *mkey;
-    /* memh and rkey_ob will be initialized on demand */
+    /* memh and rkey_ob will be initialized on demand since MD
+     * (required for memh import) is not available at this point */
```

Comment:
does it work if user opens 2 UCT MDs, and tries to use same UCT rkey on both MDs?

Reply:
> IIUC you mean use rkey with 2 different EPs which created on different ifaces + different MDs? this is a good question, I'll see how to add such test case.

Yes, this is what i've meant. Seems need a hash per md of rkey to imported key.